ui/apps/platform/src/messages/vulnMgmt.messages.ts

```diff
@@ -48,4 +48,8 @@ export const nodeScanMessages = {
         header: 'Node’s kernel unsupported.',
         body: 'This node’s kernel is not supported at this time.',
     },
+    certifiedRHELCVEsUnavailable: {
+        header: 'Unavailable or invalid CPE data',
+        body: 'The scan results are unavailable because CPE data was not found when scanning this node',
```

Comment:
Do we want to add here that the list of packages is correct but the vulnerabilities are missing?

 ```suggestion
        body: 'The list of packages is missing scan results because the data required to conduct a scan is unavailable for this node',
```

I am also not sure whether the users would know what is CPE.

Reply:
Okay